### PR TITLE
api: update progressdeadlineseconds comment for deployments

### PR DESCRIPTION
**Pull request by 0xmichalis**
@kubernetes/sig-apps-api-reviews we may never end up doing autorollback - this drops the comment from the pds field for now

### CODE REVIEW

File: pkg/apis/extensions/v1beta1/zz_generated.conversion.go [hunk 1]
```diff
@@ -346,7 +345,8 @@ func Convert_extensions_DaemonSetList_To_v1beta1_DaemonSetList(in *extensions.Da
 
 func autoConvert_v1beta1_DaemonSetSpec_To_extensions_DaemonSetSpec(in *v1beta1.DaemonSetSpec, out *extensions.DaemonSetSpec, s conversion.Scope) error {
 	out.Selector = (*v1.LabelSelector)(unsafe.Pointer(in.Selector))
-	if err := api_v1.Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec(&in.Template, &out.Template, s); err != nil {
+	// TODO: Inefficient conversion - can we improve it?
```
Comment (kargakis) [inline pkg/apis/extensions/v1beta1/zz_generated.conversion.go]: @liggitt any idea why I picked this up?

Comment (liggitt) [inline pkg/apis/extensions/v1beta1/zz_generated.conversion.go]: ummm... no?

Comment (resouer) [inline pkg/apis/extensions/v1beta1/zz_generated.conversion.go]: Just leave it alone, it's in `cmd/libs/go2idl/conversion-gen/generators/conversion.go`

**Review by resouer — COMMENTED** [review]
Also, seems need `./hack/verify-api-reference-docs.sh`

**Review by kow3ns — APPROVED** [review]

**Review by janetkuo — APPROVED** [review]

**Comment by kargakis** [comment]
rebased, reapplying lgtm

**Comment by kargakis** [comment]
@smarterclayton needs an approval

**Comment by smarterclayton** [comment]
/approve

Corrects a comment for end users and avoids confusion

**Pull request merged**